dashboard/app/models/census/ib_cs_offering.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: ib_cs_offerings
+#
+#  id          :integer          not null, primary key
+#  school_code :string(6)        not null
+#  level       :string(2)        not null
+#  school_year :integer          not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_ib_cs_offerings_on_school_code_and_school_year_and_level  (school_code,school_year,level) UNIQUE
+#
+
+class Census::IbCsOffering < ApplicationRecord
+  belongs_to :ib_school_code, foreign_key: :school_code, primary_key: :school_code, required: true
+  has_one :school, through: :ib_school_code
+
+  LEVELS = {
+    HL: "HL",
+    SL: "SL",
+  }.freeze
```

Comment:
What do these stand for? Are these common well-known abbreviations? It might be worth a comment at least.

Reply:
These are the standard level for IB CS. I added a comment to explain.